src/sdk/namespace_blob.js

```diff
@@ -215,7 +215,7 @@ class NamespaceBlob {
                         inspect(_.omit(params, 'object_md.ns')),
                         'callback res', inspect(res)
                     );
-                    return resolve(read_stream.pipe(count_stream));
+                    return resolve(read_stream);
```

Comment:
after thinking about it I think that the pass through `read_stream` is redundant. just use the `count_stream`

Reply:
removed the read_stream